akka-docs/src/main/paradox/stream/operators/Source/maybe.md

```diff
@@ -1,6 +1,6 @@
 # maybe
 
-Materialize a @scala[`Promise[Option[T]]`] @java[`CompletionStage`] that if completed with a @scala[`Some[T]`] @java[`Optional`] will emit that *T* and then complete the stream, or if completed with @scala[`None`] @java[`empty Optional`] complete the stream right away.
+Materialize a @scala[`Promise[Option[T]]`] @java[`CompletableFuture`] that if completed with a @scala[`Some[T]`] @java[`Optional`] will emit that *T* and then complete the stream, or if completed with @scala[`None`] @java[`empty Optional`] complete the stream right away.
```

Comment:
discrepancy between Scala with `Some[T]` and then no type parameter for Java. I'm thinking the type param isn't really the interesting part here though so maybe instead `Some(t)` for Scala and `Optional containing t` for Java?